File: AMS_产品管理组件/code/client/source/YssProductInfo.Support/DataCopy/Pojo/CopyDataCheck.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using FAST.Common.Service.Pojo.Base;
using Newtonsoft.Json;

namespace YssProductInfo.Support.DataCopy.Pojo
{
    /// <summary>
    /// STORY64614富国基金-【运维】产品参数复制功能优化，需考虑重复复制的情况
    /// add by zhaijiajia 20190222
    /// </summary>
    public class CopyDataCheck : BasePojo
    {
        /// <summary>
        /// 源组合
        /// </summary>
        private string c_SOURCE_CODE;

        /// <summary>
        /// 目标组合
        /// </summary>
        private string c_PORT_CODE;

        /// <summary>
        /// 数据项名称
        /// </summary>
        private string c_DATA_NAME;

        /// <summary>
        /// 数据项代码
        /// </summary>
        private string c_DATA_CODE;

        /// <summary>
        /// 复制状态
        /// </summary>
        private string c_COPY_STATE;

        /// <summary>
        /// 源组合
        /// </summary>
        [JsonProperty(PropertyName = "c_SOURCE_CODE")]
        public string C_SOURCE_CODE
        {
            get { return c_SOURCE_CODE; }
            set { c_SOURCE_CODE = value; }
        }

        /// <summary>
        /// 目标组合
        /// </summary>
        [JsonProperty(PropertyName = "c_PORT_CODE")]
        public string C_PORT_CODE
        {
            get { return c_PORT_CODE; }
            set { c_PORT_CODE = value; }
        }

        /// <summary>
        /// 数据项名称
        /// </summary>
        [JsonProperty(PropertyName = "c_DATA_NAME")]
        public string C_DATA_NAME
        {
            get { return c_DATA_NAME; }
            set { c_DATA_NAME = value; }
        }

        /// <summary>
        /// 数据项代码
        /// </summary>
        [JsonProperty(PropertyName = "c_DATA_CODE")]
        public string C_DATA_CODE
        {
            get { return c_DATA_CODE; }
            set { c_DATA_CODE = value; }
        }

        /// <summary>
        /// 复制状态
        /// </summary>
        [JsonProperty(PropertyName = "c_COPY_STATE")]
        public string C_COPY_STATE
        {
            get { return c_COPY_STATE; }
            set { c_COPY_STATE = value; }
        }
    }
}

File: AMS_产品管理组件/code/server/source/com.yss.ams.product.information.support/src/main/java/com/yss/ams/product/information/support/modules/dataCopy/pojo/CopyDataCheck.java
package com.yss.ams.product.information.support.modules.dataCopy.pojo;

import com.yss.framework.api.common.co.BasePojo;

public class CopyDataCheck extends BasePojo {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	/// <summary>
    /// 源组合
    /// </summary>
    private String c_SOURCE_CODE;

    /// <summary>
    /// 目标组合
    /// </summary>
    private String c_PORT_CODE;

    /// <summary>
    /// 数据项名称
    /// </summary>
    private String c_DATA_NAME;

    /// <summary>
    /// 数据项代码
    /// </summary>
    private String c_DATA_CODE;

    /// <summary>
    /// 复制状态
    /// </summary>
    private String c_COPY_STATE;

	public String getC_SOURCE_CODE() {
		return c_SOURCE_CODE;
	}

	public void setC_SOURCE_CODE(String c_SOURCE_CODE) {
		this.c_SOURCE_CODE = c_SOURCE_CODE;
	}

	public String getC_PORT_CODE() {
		return c_PORT_CODE;
	}

	public void setC_PORT_CODE(String c_PORT_CODE) {
		this.c_PORT_CODE = c_PORT_CODE;
	}

	public String getC_DATA_NAME() {
		return c_DATA_NAME;
	}

	public void setC_DATA_NAME(String c_DATA_NAME) {
		this.c_DATA_NAME = c_DATA_NAME;
	}

	public String getC_DATA_CODE() {
		return c_DATA_CODE;
	}

	public void setC_DATA_CODE(String c_DATA_CODE) {
		this.c_DATA_CODE = c_DATA_CODE;
	}

	public String getC_COPY_STATE() {
		return c_COPY_STATE;
	}

	public void setC_COPY_STATE(String c_COPY_STATE) {
		this.c_COPY_STATE = c_COPY_STATE;
	}
}
